### DISCUSSION TITLE
BPCells download issue

### DISCUSSION THREAD

**Discussion by Tzu-H** [satijalab/seurat#7638] [Q&A]
Despite successfully downloaded hdf5, R studio is not able to install the BPCells package using the code provided on Github. Does anyone know another way to install the package in R studio 2023.06.0+421. I have tried downloading in earlier versions, but sadly was also unsuccessful.

**Comment by sl900** [comment]
Usually a missing dependency (some library that you have to install from the terminal) that you have to manually install. Sometimes it's a long chain but once you succesfully resolve all basic dependencies Rstudio can automatically install R package dependencies for you. What is it missing is in the error it throws. Need more info (OS, and error msg)
